### DISCUSSION TITLE
Ability to pass arguments to InstallCommand

### DISCUSSION THREAD

**Discussion by abishekrsrikaanth** [spatie/laravel-package-tools#88] [Ideas]
@freekmurze, is there a way to pass arguments or inputs to the `InstallCommand`? I want to be able to change the install command based on the argument being passed. For example, if a package had to publish migrations specific to a payment provider.

```bash
php artisan package_name:install --provider=stripe
or
php artisan package_name:install --provider=paddle
```

It would help to only publish migrations specific to that condition. Or if there was a way to prompt the user for arguments in the Install Command, that would be great.

**Comment by freekmurze** [comment]
Feel free to PR this. Make sure to include tests and an updated readme.
